tensorflow/lite/testing/op_tests/concat.py

```diff
@@ -33,20 +33,31 @@ def make_concat_tests(options):
       "axis": [0, 1, 2, 3, -3, -2, -1],
       "type": [tf.float32, tf.uint8, tf.int32, tf.int64],
       "fully_quantize": [False],
+      "quantize_mode_16x8": [False],
       "dynamic_range_quantize": [False],
   }, {
       "base_shape": [[1, 3, 4, 3], [3, 4], [2, 3, 4, 3]],
       "num_tensors": [1, 2, 3, 4, 5, 6],
       "axis": [1, 2, 3, -3, -2, -1],
       "type": [tf.float32],
       "fully_quantize": [True],
-      "dynamic_range_quantize": [False],
+      "quantize_mode_16x8": [False],
+      "dynamic_range_quantize": [False]
+  }, {
+      "base_shape": [[1, 3, 4, 3], [3, 4], [2, 3, 4, 3]],
+      "num_tensors": [1, 2, 3, 4, 5, 6],
+      "axis": [1, 2, 3, -3, -2, -1],
+      "type": [tf.float32],
+      "fully_quantize": [True],
+      "quantize_mode_16x8": [True],
+      "dynamic_range_quantize": [False]
   }, {
       "base_shape": [[1, 3, 4, 3]],
       "num_tensors": [6],
       "axis": [1],
       "type": [tf.float32],
       "fully_quantize": [False],
+      "quantize_mode_16x8": [True],
```

Comment:
We want this to be false here right? Since this flag should only be true if fully_quantize is true and this test case is testing dynamic range quantize